users/views.py

```
@@ -79,4 +83,61 @@ def post(self, request, *args, **kwargs):
+            favorite.save()
+            
+            serializer = self.get_serializer(favorite)
+            return Response({
+                "status": 201,
+                "message": "즐겨찾기 추가 성공",
+                "data": serializer.data
+            }, status=status.HTTP_201_CREATED)
+
+        except Company.DoesNotExist:
+            return Response({"message": "존재하지 않는 기업 종목코드입니다."}, status=status.HTTP_404_NOT_FOUND)
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)